core/templates/components/forms/schema-based-editors/schema-based-editor.component.spec.ts

```diff
@@ -59,7 +59,7 @@ describe('Schema based editor component', function() {
     component.registerOnTouched();
 
     expect(component).toBeDefined();
-    expect(component.validate(null)).toEqual({});
+    expect(component.validate(null)).toEqual(null);
```

Comment:
Just to check, why did this change?

Reply:
Null was a better default when compared to `{}` for this case.